### ISSUE TITLE
2022.7.0 media_player.samsungtv_tizen - Requirements for samsungtv_tizen not found: ['numpy==1.21.1'].

### ISSUE THREAD

**Issue by TGMivo** [jaruba/ha-samsungtv-tizen#184]
After update HA core 2022.7.0 I got an error and not working plugin.

Platform error media_player.samsungtv_tizen - Requirements for samsungtv_tizen not found: ['numpy==1.21.1'].

**Comment by CAP-Team** [comment]
Probably because of the Python version update.

**Comment by Trebacz** [comment]
Same here just commented out of configuration.yaml for right now...

**Comment by pr0fetul** [comment]
Is this actually still maintained? Because I see a lot of issues and no actual fixes other than workarounds... I have the same issue as above. 
```
Logger: homeassistant.config
Source: config.py:873
First occurred: 8:59:48 AM (1 occurrences)
Last logged: 8:59:48 AM

Platform error: media_player - Requirements for samsungtv_tizen not found: ['numpy==1.21.1'].
```
```
Logger: homeassistant.components.hassio
Source: components/hassio/__init__.py:682
Integration: Home Assistant Supervisor (documentation, issues)
First occurred: 9:34:12 AM (1 occurrences)
Last logged: 9:34:12 AM

The system cannot restart because the configuration is not valid: Platform error media_player.samsungtv_tizen - Requirements for samsungtv_tizen not found: ['numpy==1.21.1'].
```

**Comment by daniperaleda** [comment]
Hello, Same error in my installation. I have tried to reinstall it in linux and it seemed to be installed. But doesn't fix the problem. Please, help to fix it...I have based a lot of my TV configuration on it

**Comment by TGMivo** [comment]
I used a backup from before 2022.7.0 and it fixed for now…

**Comment by daniperaleda** [comment]
Downgraded to 2022.6. Waiting to the confirmation when fixed to upgrade again

**Comment by Chicayban** [comment]
wo

> Its simple to repair: config\custom_components\samsungtv_tizen -> manifest.json
> 
> change
> 
> "numpy==1.21.1" to "numpy==1.23.0"
> 
> After change this restart HA, next add config to YAML.

works for me too

**Comment by ruttromp** [comment]
@jaruba please review PR #185 to fix this issue. Thanks.

**Comment by nogueirasa** [comment]
@jaruba , we love your integration, please, don´t give it away

**Comment by iWelna** [comment]
yea @jaruba pls update :) we love your integration

**Comment by jaruba** [comment]
Sorry for the long wait, work and maintaining many projects can take its toll

I just released v1.6.0 with an updated numpy version

**Comment by iWelna** [comment]
Thanks a lot !
